### PR TITLE
NONAME-179 standardized pydentic model validation

### PR DESCRIPTION
**Pull request by yanvoi**

### CODE REVIEW

File: backend/src/api/models/users/user.py [hunk 1]
```diff
@@ -9,23 +7,16 @@ class User(Base):
...
-    def validate_users(cls, v, info: ValidationInfo) -> list[str]:
-        for email in v:
-            if not re.match(r"^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$", email):
-                raise ValueError("invalid email format")
-        return v
+    def validate_users(cls, v) -> list[str]:
+        return Base.validate_emails(v)
 
 
 key_pem_regex = r"-----BEGIN PUBLIC KEY-----(\n|\r|\r\n)([0-9a-zA-Z\+\/=]{64}(\n|\r|\r\n))*([0-9a-zA-Z\+\/=]{1,63}(\n|\r|\r\n))?-----END PUBLIC KEY-----"
```
Comment (mtszlachetka) [inline backend/src/api/models/users/user.py:22]: this can be deleted now

Comment (yanvoi) [reply backend/src/api/models/users/user.py]: Done

**Review by mtszlachetka — APPROVED** [review]
Good stuff

**Review by K4RP4T — APPROVED** [review]
awesome

**Pull request merged**